package.json

```diff
@@ -3,14 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "enzyme-to-json": "^3.3.5",
```

Comment:
@rkemmy, let's have this as a devDependency and not production dependency as currently is. 🙂

Reply:
I have changed it but out of curiosity, in future, how do I identify which dependencies should be in production and which shouldn't?

Reply:
@rkemmy, if a dependency is only useful during development work such as testing dependencies and package bundling dependencies like webpack, babel dependencies, code syntax linters etc, then it's a development dependency. Else if a dependency will be needed for the application to be able to run at all in a production environment, e.g. react itself, redux, then it's a production dependency. 

I think this can shed some more light https://stackoverflow.com/a/46060407/6951110. But you will slowly start to find it intrinsic as you interact with dependencies in a project. Just ask yourself, 'Do we need this dependency post-development?' If no, it's a devDependency.